src/apps/HeyGPT/HeyGPT/ViewModels/LetsChatViewModel.cs

```diff
@@ -62,26 +77,28 @@ private async Task OnExecuteSendMessageCommand()
     {
         var userMessage = UserMessage?.Trim() ?? string.Empty;
 
-        AddUserMessageToHistory(userMessage);
+        await SendMessageInternal(userMessage);
```

Comment:
Previously, the only place we sent the messages was from the Command.  But now we have the protocol and file handlers that also can send messages.  So we do a crude breakout of the functionality into a private method.